gateway/gateway.go

```diff
@@ -443,7 +459,9 @@ func (gateway *HandleT) userWebRequestWorkerProcess(userWebRequestWorker *userWe
 		for _, req := range breq.batchRequest {
 			writeKey := req.writeKey
 			sourceTag := gateway.getSourceTagFromWriteKey(writeKey)
```

Comment:
Do we still need this tag now that we are using sourceID?

Reply:
This is used in grafana dashboards now. Let's keep it for now.